File: src/main/java/com/gary/Hello.java
package com.gary;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new Person().Hello();
        Person p = new Person("Tom", 66.5f, 1.7f);
        p.Hello();
        System.out.println( p.bmi() );
    }
}

File: src/main/kotlin/com/gary/Hello.kt
package com.gary

fun main() {
//    println("Hello kotlin")
//    Human().hello()
    val h = Human( 66.5f, 1.7f )
    println( h.bmi() )

//    val age = 12
//    var age2 : Int = 13;
//    age2 = age2.plus( age )
//    println( age2 )

}

class Human(var weight: Float, var height: Float) {
    init {
        println("test $weight")
    }
    fun hello(){
        println("Hello kotlin")
    }
    fun bmi() : Float{
        var bmi = weight / ( height * height)
        return bmi;
    }
}